#1	----------------------------------------------------------------------
PANIC: unprotected error in call to Lua API ([string "error ('catch error')"]:1: catch error)
XFAIL test-libncluaw-at-panic-x (exit status: 134)
